src/IntelliTect.Coalesce/TypeDefinition/ReflectionRepository.cs

```diff
@@ -163,6 +163,28 @@ private void ProcessAddedType(TypeViewModel type)
             _generatedParamDtos[cvm] = type.ClassViewModel!;
         }
 
+        // Check for SimpleModelAttribute first to treat as Simple Model
+        if (type.HasAttribute<SimpleModelAttribute>())
+        {
+            if (_rootTypeWhitelist != null && !_rootTypeWhitelist.Contains(type.Name))
+            {
+                return;
+            }
+
+            if (type.ClassViewModel != null)
+            {
+                // Null this out so it gets recomputed on next access.
+                _clientTypes = null;
+                
+                var classViewModel = type.ClassViewModel;
+                if (_externalTypes.Add(classViewModel))
+                {
+                    DiscoverExternalPropertyTypesOn(classViewModel);
+                }
+            }
+            return;
+        }
+
```

Comment:
This new attribute must be used alongside [Coalesce], just like [Service] which I told you to model this off of.